### ISSUE TITLE
No autocomplete for Vec<(i32, i32)>::resize method

### ISSUE THREAD

**Issue by dima74** [intellij-rust/intellij-rust#4223]
## Environment

* **IntelliJ Rust plugin version:** 0.2.102.2130-192
* **Rust toolchain version:** rustc 1.36.0 (a53f9df32 2019-07-03)
* **IDE name and version:** CLion 2019.2 Build #CL-192.5728.100
* **Operating system:** Linux

## Problem description
Autocomplete for [`resize`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.resize) method of `Vec<(i32, i32)>` doesn't show up:

![image](https://user-images.githubusercontent.com/6505554/62422547-f1e60000-b6bc-11e9-9f04-a5b849ffb045.png)

However, `resize` method exists and [can be used](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=13ab32c3cc5bef9d092df65f1a41242c).

## Steps to reproduce
Press Ctrl+Space in the following code:
```rust
fn main() {
    let mut a: Vec<(i32, i32)> = Vec::new();
    a.resi<cursor>
}
```

**Comment by Kobzol** [comment]
I looked briefly into this and I think that `Clone` is not inferred for `(i32, i32)` and that's why the method is not offered.

**Comment by Kobzol** [comment]
This is caused by the fact that `Clone` and `Copy` is not inferred for `(u32, u32)`.

**Comment by Kobzol** [comment]
I believe that this issue has been fixed with https://github.com/intellij-rust/intellij-rust/pull/4741 and can be closed.

**Comment by Undin** [comment]
@Kobzol thanks!

Fixed in #4741
